### ISSUE TITLE
Ability to fetch full article in-app

### ISSUE THREAD

**Issue by BobIsMyManager** [bubelov/news#133]
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [X] This issue contains only one feature reqest. I will open one issue for every feature I want to request.

#### What feature do you want?

I want to be able to load the full article in app, because some feeds just say what the article is about and tell you to go to the actual article, like [OMG! Ubuntu!](https://omgubuntu.co.uk/feed), [Ionic](https://ionicframework.com/blog/feed/), and [ItsFOSS](https://itsfoss.com/feed/)

#### Why do you want this feature?

Because i like the apps's UI and would rather read these feeds in app than having to open a web browser to read them with its less than ideal UI

**Comment by JohnFrankz** [comment]
Yes, if this app could add the option to get the full text of Rss, then it would be the perfect Android Rss reading in my mind, its nice and simple interface has impressed me deeply.

**Comment by Kvasenok** [comment]
In most cases you can use feed expansion services like feedex or fivefilters to have a full text feed.

**Comment by Darkempire78** [comment]
News will be perfect with this improvement

**Comment by Moe23333** [comment]
I need this feature too! [Feeder](https://gitlab.com/spacecowboy/Feeder) has this feature, but I prefer News app. I'd like to see News app also add this feature.

**Comment by d4f5409d** [comment]
+1 Helper comment, I want this too!

**Comment by bubelov** [comment]
Embedded backend is not the main focus since I only use and test Miniflux, so without dedicated maintainers those pretty involved features are impossible. Miniflux does expand some feeds though, and any expansion service should be compatible with embedded
